src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -11,8 +11,8 @@ import cromwell.engine.db.DataAccess._
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.WorkflowActor.{Restart, Start}
+import cromwell.engine.workflow.WorkflowManagerActor.{WorkflowManagerData, WorkflowManagerState}
```

Comment:
These two are actually used as type variables in the *definition* of the class.  But I can `_` this and get rid of the other import below.